.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ env:
   RSYNC_HOST: ijzer
 jobs:
   build-linux:
-    runs-on: [linux]
+    runs-on: [ubuntu-latest]
     container: ghcr.io/watem-sedem/ci/watem-sedem-conda
     env:
       PYTHONPATH: ../CNWS-processing-scripts/pycnws/src
```

Comment:
env python path -> ../CNWS-processing-scripts/pycnws/src -> can this be renamed to ../erosion-sediment-model/pywatemsedem/src